### ISSUE TITLE
feat: update relay-angular-chat example

### ISSUE THREAD

**Issue by weboko** [waku-org/js-waku-examples#247]
### Problem

This example is too old and it is hard to update `@waku/*` version to latest due to problem with build that appears. 

### Solution 

Update `angular` related dependencies **or** eject webpack config and make it work with latest `@waku/*`.

### Notes

Ref PR: https://github.com/waku-org/js-waku-examples/pull/246
Original issue: https://github.com/waku-org/js-waku/issues/1353

**Comment by fryorcraken** [comment]
I suggest we create a bounty for someone to create a small example that uses Angular.
As part of the bounty, we can set some criteria to make long term maintenance easier (if such criteria can exist)

**Comment by weboko** [comment]
> I suggest we create a bounty for someone to create a small example that uses Angular. As part of the bounty, we can set some criteria to make long term maintenance easier (if such criteria can exist)

good idea, I will post an issue for bounty but I would consider this bounty low priority since we have other examples and angular is not that popular

**Comment by fryorcraken** [comment]
Let's update the example in any case and we can have a guide for it at some point cc @LordGhostX 

For the bounty, tracked here: https://github.com/waku-org/bounties/issues/2
